src/test/test_hs_client.c

```diff
@@ -1007,6 +1029,104 @@ test_close_intro_circuits_new_desc(void *arg)
   UNMOCK(networkstatus_get_live_consensus);
 }
 
+static void
+test_socks_hs_errors(void *arg)
+{
+  int ret;
+  char *desc_encoded = NULL;
+  ed25519_keypair_t service_kp;
+  ed25519_keypair_t signing_kp;
+  entry_connection_t *socks_conn = NULL;
+  dir_connection_t *dir_conn = NULL;
+  hs_descriptor_t *desc;
```

Comment:
This being non-NULL causes compilation errors that break travis.

Reply:
Fixed in the code.